web/services/createTemplatesService.php

```diff
@@ -27,8 +27,6 @@
 
 $request = trim(file_get_contents('php://input'));
 
-/*$request = '<?xml version="1.0" encoding="UTF-8"?><templates><template><customerId></customerId><projectId>1</projectId><ttype>community</ttype><story>wofff</story><taskStoryId></taskStoryId><telework>true</telework><onsite></onsite><text>wofff</text></template><template><customerId></customerId><projectId>1</projectId><ttype></ttype><story>asdf</story><taskStoryId></taskStoryId><telework>false</telework><onsite>true</onsite><text>woaowa</text></template><template><customerId></customerId><projectId>1</projectId><ttype></ttype><story>waowao</story><taskStoryId></taskStoryId><telework>true</telework><onsite></onsite><text>asdf</text></template></templates>';
-*/
```

Comment:
I'd rather leave this line :innocent:: we have them in most service pages, and they serve as examples of the kind of XML that the service admits. Sorry it's a bit obscure; we should better document these services.

Reply:
Ok I won't fight for this one yet cause we don't have automated tests! hehe as soon as we do I would argue to remove all of these commented lines

Reply:
commit removed